packages/plugins/image/src/renderer.tsx

```diff
@@ -50,22 +50,17 @@ const Pending = styled.div({
 export class ImageRenderer extends React.Component<ImageRendererProps> {
   public render() {
     const { state, disableMouseEvents } = this.props
+    const alt = state.alt.defined ? state.alt.value : ''
```

Comment:
@inyono Is there an easy way to get a string from a text plugin state?

In any case: If not I'll hack something up in the frontend to make sure we have an alt tag wherever possible, but I'm not sure how important this is here.

Reply:
[slate-plain-serializer](https://docs.slatejs.org/v/v0.47/other-packages/index-2) might be what you are looking for.

Reply:
Thanks I'll investigate!